### PR TITLE
[T.F 2.0 API Docs] Adding desc and example to math functions

### PR DESCRIPTION
**Pull request by SSaishruthi**
This PR reference #25802

Added examples and explanation to:

1. cos
2. cosh
3. sin
4. sinh
5. tan
6. tanh

Thanks in advance

### CODE REVIEW

File: tensorflow/core/api_def/base_api/api_def_Cos.pbtxt [hunk 1]
```diff
@@ -1,4 +1,13 @@
 op {
   graph_op_name: "Cos"
   summary: "Computes cos of x element-wise."
+  description: << END
+  Given an input tensor, this function computes cosine of every
+  element in the tensor. Input range is (-infinity, infinity) and
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_Cos.pbtxt]: backticks around ranges (all files), please infinity -> inf

File: tensorflow/core/api_def/base_api/api_def_Cos.pbtxt [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+  output range is [-1,1].
+  ```python
+  x = tf.constant([-9, -0.5, 1, 1.2, 200, 10000])
+  tf.math.cos(x, name ='cos') ==> [-0.91113025  0.87758255  0.5403023   0.3623577   0.48718768 -0.95215535]
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_Cos.pbtxt]: remove name arg

**Review by martinwicke — CHANGES_REQUESTED** [review]
I stopped commenting -- please apply comments across all files (and PRs)

**Comment by SSaishruthi** [comment]
Need some more update. Will push changes soon.

**Comment by SSaishruthi** [comment]
@martinwicke Completed changes. Please let me know if this looks good.

**Comment by SSaishruthi** [comment]
@martinwicke Completed changes. Please let me know if this looks good.

File: tensorflow/core/api_def/base_api/api_def_Cos.pbtxt [hunk 3]
```diff
@@ -1,4 +1,15 @@
...
   graph_op_name: "Cos"
   summary: "Computes cos of x element-wise."
+  description: << END
+  Given an input tensor, this function computes cosine of every
+  element in the tensor. Input range is `(-inf, inf)` and
+  output range is `[-1,1]`. If input lies outside the boundary, `nan`
+  is returned.
+
+  ```python
+  x = tf.constant([-float("inf"), -9, -0.5, 1, 1.2, 200, 10000, float("inf")])
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_Cos.pbtxt]: Should be -0.9, not -9 I think. Please check.

Comment (martinwicke) [reply tensorflow/core/api_def/base_api/api_def_Cos.pbtxt]: Never mind.

**Review by martinwicke — DISMISSED** [review]

**Comment by SSaishruthi** [comment]
> @SSaishruthi Can you please check build failures? Thanks!

Hi @gbaned 

I noticed the error in log files.
```
Execution platform: @org_tensorflow//third_party/toolchains:rbe_cuda10.0-cudnn7-ubuntu14.04
2019-06-01 04:07:50.987236: F tensorflow/python/framework/python_op_gen_main.cc:123] Non-OK-status: api_def_map.LoadFileList(env, api_files) status: Invalid argument: Error parsing ApiDef file tensorflow/core/api_def/base_api/api_def_Sinh.pbtxt: 5(1): Expected identifier, got:
```

I am not sure where the error is as the format is similar to other file that got merged recently.
Also, there is an open issue related to this.
https://github.com/tensorflow/tensorflow/issues/28871

Can you please help me with this?

**Comment by SSaishruthi** [comment]
Hi @gbaned 

Is everything fine with this PR?

**Review by martinwicke — APPROVED** [review]

**Comment by SSaishruthi** [comment]
Hi @martinwicke 

Last time it was throwing an error in `sinh` and this time in `cosh`. Not sure what is happening here.
Can you please help?

**Comment by martinwicke** [comment]
That is pretty mysterious, I can't find anything wrong with the syntax. @josh11b does this look right to you? I says it's expecting an identifier, but as far as I can tell. the `<<END` thing is done correctly.

**Comment by SSaishruthi** [comment]
Hi @gbaned @martinwicke 

Is there anything to be modified from my side? I some made some similar doc updates but not sure if it is ok to create PR without resolving this issue. Please suggest.

File: tensorflow/core/api_def/base_api/api_def_Cos.pbtxt [hunk 4]
```diff
@@ -1,4 +1,15 @@
 op {
   graph_op_name: "Cos"
   summary: "Computes cos of x element-wise."
+  description: << END
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_Cos.pbtxt]: All of these need to look like this: `description: <<END`, *without* the space.

**Review by martinwicke — DISMISSED** [review]
Found what's wrong. Should go in once fixed.

File: tensorflow/core/api_def/base_api/api_def_Cos.pbtxt (hunk 4 above)
Comment (SSaishruthi) [reply tensorflow/core/api_def/base_api/api_def_Cos.pbtxt]: Thanks a lot, @martinwicke 
I have updated the code

Comment (SSaishruthi) [reply tensorflow/core/api_def/base_api/api_def_Cos.pbtxt]: Do we need to trigger the test again?

**Review by martinwicke — APPROVED** [review]

**Pull request merged**